setup.py

```diff
@@ -11,20 +11,19 @@
 if sys.version_info[0] == 3:
     mysql_req = "mysqlclient>=1.3.6"
 else:
-    mysql_req = "MySQL-python>=1.2.2"
+    mysql_req = "MySQL-python>=1.2.5"
 
 
 setup(
     name='django-mysqlndb-backend',
-    version='1.1.0',
+    version='1.2.0',
     author_email='programmers@theatlantic.com',
     packages=['mysqlndb'],
     url='https://github.com/theatlantic/django-mysqlndb-backend',
     description='Provides a django database backend that works with MySQL Cluster\'s NDB storage engine.',
     install_requires=[
-        "Django >= 1.4",
+        "Django >= 1.11",
```

Comment:
I don't think we want to change this to `>= 1.11`, since this change isn't backwards incompatible. That way we can upgrade this package before we do the final 1.11 upgrade